### PR TITLE
Added destination node to channels, added new methods to channels repo

### PR DESCRIPTION
**Pull request by Jossec101**
Added destination node to channels, added new methods to channels repository

### CODE REVIEW

File: src/Data/Repositories/ChannelRepository.cs [hunk 1]
```diff
@@ -22,8 +22,12 @@
 using FundsManager.Data.Repositories.Interfaces;
 using FundsManager.Jobs;
 using FundsManager.Helpers;
-using Quartz;
+ using FundsManager.Services;
+ using Grpc.Core;
+ using Lnrpc;
+ using Quartz;
 using Microsoft.EntityFrameworkCore;
+ using Channel = FundsManager.Data.Models.Channel;
```
Comment (RodriFS) [inline src/Data/Repositories/ChannelRepository.cs:30]: I don't know why Rider adds this space before the "using", it happens to me as well

File: src/Data/Repositories/ChannelRepository.cs [hunk 2]
```diff
@@ -175,5 +179,97 @@ public async Task<(bool, string?)> SafeRemove(Channel type, bool forceClose = fa
...
+        {
+            await using var applicationDbContext = await _dbContextFactory.CreateDbContextAsync();
+            
+            //check the channel exists in the node
+            var channelOperationRequest = channel.ChannelOperationRequests.FirstOrDefault();
+            
+            if (channelOperationRequest == null)
+            {
+                _logger.LogError("Error while marking as closed with id: {ChannelId}. No channel operation request found", channel.Id);
+                return (false, "No channel operation request found");
```
Comment (RodriFS) [inline src/Data/Repositories/ChannelRepository.cs:248]: This is a very Golang thing to do, why don't we use custom exceptions?

**Review by RodriFS — APPROVED** [review]

File: src/Data/Repositories/ChannelRepository.cs (hunk 2 above)
Comment (Jossec101) [reply src/Data/Repositories/ChannelRepository.cs:248]: I took it from Golang long time ago, true 👯 . I mean exceptions are cool but they need to be try-catched and sometimes they are a pain in performance. The main problem/feature is that all of the repositories have a golang-based error approach.

**Review by RodriFS — APPROVED** [review]

File: src/Data/Repositories/ChannelRepository.cs (hunk 1 above)
Comment (markettes) [reply src/Data/Repositories/ChannelRepository.cs:30]: Happening to me too. It is a pain in the ***

**Pull request merged**